### PR TITLE
Feature/implement UI for countdown clock

### PR DESCRIPTION
**Pull request by nddmanh**
UI countdown:
image

### CODE REVIEW

**Comment by ppppbn** [comment]
Again, refactor your commits

File: src/modules/playRoom/components/CountDown.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const CountDown = () => {
+  return (
+    <div className="countdown">
+      <p id="time-countdown">
```
Comment (ppppbn) [inline src/modules/playRoom/components/CountDown.js:6]: Why do you need id here?

File: src/modules/playRoom/pages/PlayRoom.css [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    width: 100%;
+    height: 350px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 10px;
+    border: solid 5px #77b3d4
+}
+
+#time-countdown {
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.css:190]: Don't write CSS with id

File: src/modules/playRoom/pages/PlayRoom.css [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    position: relative;
+}
+
+/* scroll bar  */
+.scroll {
+    width: 100%;
+    height: 74%;
+    overflow: hidden;
+    overflow-y: scroll;
+    /* overflow-y: scroll; */
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.css:59]: Remove this

File: src/modules/playRoom/pages/PlayRoom.css [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+}
+
+.client-info {
+    color: #fff;
+    padding: 5px;
+    margin-top: 15px;
+}
+.client-info > h2 {
+    font-weight: 650;
+    color: white;
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.css:130]: Using hex code for color

File: src/modules/playRoom/components/ChatBox.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/ChatBox.js:1]: Remove React

File: src/modules/playRoom/components/CountDown.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/CountDown.js:1]: Remove React

File: src/modules/playRoom/components/PickItem.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:1]: Remove React

File: src/modules/playRoom/components/PickItem.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
+  return (
+    <>
+      <div class="select"  onClick={showModal} >
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:22]: Do not named a common class name it can make mistake

Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:22]:
```suggestion
      <div className="select" onClick={showModal} >                
```

File: src/modules/playRoom/components/PickItem.js [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
+  return (
+    <>
+      <div class="select"  onClick={showModal} >                
+        <input type="image" className="img-select" src={img.src} alt="" name="" value={img.value} />
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:23]: Do not leave attributes empty

File: src/modules/playRoom/components/PickList.js [hunk 10]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickList.js:1]: Remove React

File: src/modules/playRoom/components/PickList.js [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+import { Row, Col } from 'antd';
+import PickItem from './PickItem';
+
+const PickList = () => {
+  const [imagesPick, setImagesPick] = useState([
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickList.js:6]: Move ```imagesPick``` default value to a constant file

File: src/modules/playRoom/pages/PlayRoom.css [hunk 12]
```diff
@@ -0,0 +1,192 @@
+
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:1]: We use ```scss``` in this project, not ```css```

File: src/modules/playRoom/pages/PlayRoom.css [hunk 13]
```diff
@@ -0,0 +1,192 @@
...
+#game {
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:2]: Do not use id to style an element

File: src/modules/playRoom/pages/PlayRoom.js [hunk 14]
```diff
@@ -1,8 +1,23 @@
 import React, { useState, useEffect } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.js:1]: Remove React, useState, useEffect

**Pull request closed without merging**